velox/connectors/hive/HiveConfig.cpp

```diff
@@ -243,6 +243,17 @@ bool HiveConfig::s3UseProxyFromEnv() const {
   return config_->get<bool>(kS3UseProxyFromEnv, false);
 }
 
+uint8_t HiveConfig::parquetReadTimestampUnit(const Config* session) const {
+  const auto unit = session->get<uint8_t>(
+      kParquetReadTimestampUnitSession,
+      config_->get<uint8_t>(kParquetReadTimestampUnit, 9 /*nano*/));
+  VELOX_CHECK(
+      unit == 0 /*second*/ || unit == 3 /*milli*/ || unit == 6 /*micro*/ ||
```

Comment:
Do we have the option to provide truncate to second?  It seems only options available in Presto is microseconds or milliseconds

Reply:
Thanks for the catch. Updated.